modules/installation-azure-user-infra-uploading-rhcos.adoc

```diff
@@ -58,17 +74,37 @@ less than or equal to the {product-title} version that you install. Use the imag
 that matches your {product-title} version if it is available.
 ====
 
-. Copy the chosen VHD to a blob:
+. Create the storage container for the VHD:
 +
 [source,terminal]
 ----
 $ az storage container create --name vhd --account-name ${CLUSTER_NAME}sa --account-key ${ACCOUNT_KEY}
 ----
+ifdef::ash[]
+. Download the compressed {op-system} VHD file locally:
++
+[source,terminal]
+----
+$ curl -O -L ${COMPRESSED_VHD_URL}
+----
+
+. Decompress the VHD file.
+endif::ash[]
+
+. Copy the chosen VHD to a blob:
 +
+ifdef::azure[]
 [source,terminal]
 ----
 $ az storage blob copy start --account-name ${CLUSTER_NAME}sa --account-key ${ACCOUNT_KEY} --destination-blob "rhcos.vhd" --destination-container vhd --source-uri "${VHD_URL}"
 ----
+endif::azure[]
+ifdef::ash[]
+[source,terminal]
+----
+$ az storage blob upload --account-name ${CLUSTER_NAME}sa --account-key ${ACCOUNT_KEY} -c vhd -n "rhcos.vhd" -f rhcos-<rhcos_version>-azurestack.x86_64.vhd
+----
+endif::ash[]
 +
 To track the progress of the VHD copy task, run this script:
```

Comment:
you can delete this part about tracking the storage blob copy. 

We are doing a `storage blob upload` which already has a progress bar.